runtime/frequency/src/lib.rs

```diff
@@ -344,7 +344,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("frequency-testnet"),
 	impl_name: create_runtime_str!("frequency"),
 	authoring_version: 1,
-	spec_version: 82,
+	spec_version: 83,
```

Comment:
why do we have to increase?

Reply:
So everytime we change the code inside runtime (wasm) we would need to increase this. The reason is **seamless runtime upgrades**. For nodes to upgrade the version of their runtime, we should increase it after any change so that the nodes figure out that their version is an older one and they have to upgrade to the newer one.